bin/compile

```diff
@@ -211,7 +211,7 @@ restore_cache | output "$LOG_FILE"
 build_dependencies() {
   local cache_status start
 
-  run_if_present 'heroku-prebuild'
+  run_if_present "$BUILD_DIR" 'heroku-prebuild'
```

Comment:
Adding the directory as an input to these functions is necessary for addressing https://github.com/heroku/heroku-buildpack-nodejs/issues/385 anyway